RP-1.netkan

```diff
@@ -17,7 +17,6 @@
         { "name" : "CustomBarnKit" },
         { "name" : "DeadlyReentry" },
         { "name" : "VenStockRevamp" },
-        { "name" : "CommunityTechTree" },
         { "name" : "KerbalConstructionTime" },
         { "name" : "RealSolarSystem" }
     ],
```

Comment:
Add following mods to the depends according to the readme in #864 
```
{ "name" : "ProceduralParts" },
{ "name" : "DMagicOrbitalScience" }
```

Remove in recommends
```
{ "name" : "ProceduralParts" },
```

Rename in recommends
`RCSBuildAid` to `RCSBuildAidCont`
`HangarExtender` to `HangerExtenderExtended`
`TextureReplacer` to `TextureReplacerReplaced`